src/main/java/com/pinterest/secor/uploader/HadoopS3UploadManager.java

```diff
@@ -63,7 +82,14 @@ public HadoopS3UploadManager(SecorConfig config) {
             @Override
             public void run() {
                 try {
-                    FileUtil.moveToCloud(localLogFilename, logFileName);
+                    if (customTopicsNamesMap != null) {
+                    	String logFileNameCustomTopicName = logFileName;
+                    	logFileNameCustomTopicName = logFileNameCustomTopicName.replace(topicName,
+                    			customTopicsNamesMap.get(topicName));
+                    	FileUtil.moveToCloud(localLogFilename, logFileNameCustomTopicName);
+                    }else {
```

Comment:
Coding style: need a space after }

Reply:
Thank you very much @HenryCaiHaiying . I will to improve the code to submit again.